src/pretix/plugins/stripe/migrations/0004_auto_20211208_1407.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 3.2.2 on 2021-12-08 14:07
+
+from django.db import migrations
+
+
+def cleanup(app, schema_editor):
+    EventSettingsStore = app.get_model('pretixbase', 'Event_SettingsStore')
+    EventSettingsStore.objects.filter(key='payment_stripe_method_cc').update(key='payment_stripe_method_card')
```

Comment:
Updating EventSettingsStore directly is dangerous since it does not clean up hierarkey's the redis-level caches. Can we access ``event.settings`` in migrations? Otherwise we'd need to clean them out manually.